spec/views/names/tabs/tabs_partial_spec.rb

```
@@ -0,0 +1,108 @@
+      expect(rendered).to have_link('Edit', id: 'name-edit-tab')
+    end
+  end
+
+  context "when a user can delete a name" do
+    before do
+      allow(view).to receive(:can?).with('names', 'delete').and_return(true)
+    end
+
+    it "does not render the Delete tab" do
+      subject
+      expect(rendered).to have_link('Delete name', id: 'name-delete-tab')
```

RuboCop (1.69.1)

[convention] Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.

(Style/StringLiterals)